polyglot/graalvm/README.md

```diff
@@ -0,0 +1,18 @@
+# Running Polyglot application on Payara Micro with GraalVM
+
+Project contains example of JavaScript library called from Kotlin written JAX-RS resources.
```

Comment:
This wording is a bit weird. I would rewrite as follows:

"This project is an example of mixing JavaScript, Kotlin and Java EE using GraalVM. It demonstrates calling of a JavaScript library from JAX-RS resources, which are written in Kotlin."